vllm/engine/arg_utils.py

```diff
@@ -907,6 +925,23 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             "--disable-hybrid-kv-cache-manager",
             **scheduler_kwargs["disable_hybrid_kv_cache_manager"])
 
+        logitsprocs_group = parser.add_argument_group(
+            title="Logits processors",
+            description="Logits processors settings.",
+        )
+        logitsprocs_group.add_argument(
+            "--logits-processors-entrypoints",
```

Comment:
I think it might be better to not have this arg. Just load any/all LPs that are defined in the LP entrypoints group.